### PR TITLE
Preparation for pysat 3.0.0 release

### PR DESCRIPTION
**Pull request by jklenzing**
link to pysat/pysat#676

### CODE REVIEW

**Comment by jklenzing** [comment]
Updates here need to be merged in before #11 can be merged

**Comment by jklenzing** [comment]
NOTE: a test branch has been built for github actions, but there are some issues there involving pysatMagVect and numpy that are still being sorted

File: pysatIncubator/tests/test_instruments.py [hunk 1]
```diff
@@ -46,16 +46,16 @@ def setup_class(self):
         # Make sure to use a temporary directory so that the user's setup is not
         # altered
         self.tempdir = tempfile.TemporaryDirectory()
-        self.saved_path = pysat.data_dir
-        pysat.utils.set_data_dir(self.tempdir.name, store=False)
+        self.saved_path = pysat.params['data_dirs']
+        pysat.params.data['data_dirs'] = [self.tempdir.name]
```
Comment (rstoneback) [inline pysatIncubator/tests/test_instruments.py:50]: FYI if you are setting a single path it is ok to use 
```
pysat.params.data['data_dirs'] = self.tempdir.name
```

**Review by rstoneback — APPROVED** [review]

**Comment by rstoneback** [comment]
> NOTE: a test branch has been built for github actions, but there are some issues there involving pysatMagVect and numpy that are still being sorted

I'm late with this comment, but, OMMBV is now on pip so it could be time to retire pysatMagVect. The older vectors in pysatMagVect are less correct.

**Comment by jklenzing** [comment]
I thought of that, but in terms of triage this is lower on my list. Probably worth opening an issue so we don't forget. The GA issue seems to be specific to numpy and fortran.  Seeing it in pysatMissions too.

**Pull request merged**